sherpa/astro/ui/utils.py

```diff
@@ -13618,7 +13600,8 @@ def eqwidth(self, src, combo, id=None, lo=None, hi=None, bkg_id=None,
                 covar_matrix=None):
         """Calculate the equivalent width of an emission or absorption line.
 
-        The equivalent width [1]_ is calculated in the selected units
+        The `equivalent width <http://en.wikipedia.org/wiki/Equivalent_width>`_
```

Comment:
```suggestion
        The `equivalent width <https://en.wikipedia.org/wiki/Equivalent_width>`_
```

Reply:
Thanks for checking @wmclaugh , but before I address those, I'll wait and see if we can take something like this PR at all or if it messes up the generation of ahelp files too much that @DougBurke does; there is no hurry either way.